src/core/symbology/qgsrulebasedrenderer.h

```diff
@@ -343,7 +343,7 @@ class CORE_EXPORT QgsRuleBasedRenderer : public QgsFeatureRenderer
         QSet< QString > legendKeysForFeature( QgsFeature &feat, QgsRenderContext *context = nullptr );
 
         //! tell which rules will be used to render the feature
```

Comment:
Maybe add an explanation of the `onlyActive` argument here too

Reply:
done